firstvoices/backend/search/logging.py

```diff
@@ -0,0 +1,30 @@
+import logging
```

Comment:
+1 for this change. Thank you for moving these to a separate file for them to be reusable.